__tests__/seed.test.js

```diff
@@ -0,0 +1,35 @@
+const data = require("../db/data/test-data");
+const seed = require("../db/seeds/seed")
+const db = require("../db/connection.js")
+const request = require("supertest")
+const app = require("../app.js")
+
+beforeEach(() => {
+    return seed(data)
+})
+
+afterAll(() => {
+    db.end()
+})
+
+describe("GET /api/topics", () => {
+    test("responds with an array of topic objects", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => {
+            expect(body.length > 0).toBe(true)
+            expect(body[0]).toHaveProperty('slug')
```

Comment:
this test isn't very robust since you're only checking the first topic. I would suggest iterating through the array of topics and asserting against each one, something like .toMatchObject or .objectContaining would be good to use